demo.js

```diff
@@ -0,0 +1,49 @@
+const tagManagerDataLayer = [];
+tagManager = new window.tagmanagerWrap.default(tagManagerDataLayer, {
+  gtmId: 'GTM-T2HGD',
```

Comment:
@iondrimba since this project is open source, we have to remove gtmId, right?

Reply:
@williamcosta Spot on👍

Reply:
@williamcosta GTM id is a public information. I think that is not a problem to mantain this info here, because is not a secret. Our HTML source contains this number on our pages.

Reply:
@eduardojmatos Good point 👍

Reply:
@eduardojmatos you're right.